src/app/shared/address/tf-address.component.scss

```diff
@@ -0,0 +1,12 @@
+@import './../sass/_index';
+
+li {
+  color:$black;
+}
+
+.panel-heading{
```

Comment:
for eg: This is a class name css selector. Use something like this but with `BEM css` for the above comment